ios-base/Extensions/FontExtension.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  FontExtension.swift
+//  ios-base
+//
+//  Created by Germán Stábile on 2/28/19.
+//  Copyright © 2019 TopTier labs. All rights reserved.
+//
+
+import UIKit
+import RSFontSizes
+
+extension UIFont {
+  var bold: UIFont { return withWeight(.bold) }
+  var semibold: UIFont { return withWeight(.semibold) }
+  var light: UIFont { return withWeight(.ultraLight) }
+  
+  private func withWeight(_ weight: UIFont.Weight) -> UIFont {
+    var attributes = fontDescriptor.fontAttributes
+    var traits = (attributes[.traits] as? [UIFontDescriptor.TraitKey: Any]) ?? [:]
+    
+    traits[.weight] = weight
+    
+    attributes[.name] = nil
+    attributes[.traits] = traits
+    attributes[.family] = familyName
+    
+    let descriptor = UIFontDescriptor(fontAttributes: attributes)
+    
+    return UIFont(descriptor: descriptor, size: pointSize)
+  }
+  
+  static func font(withName name: String, size: CGFloat) -> UIFont {
+    let size = Font.PointSize.proportional(to: (.screen6_5Inch,
```

Comment:
Using proportional to 6.5 inches screens since designs are mostly relative to iPhone XS